bin/noderedrevpinodes-server.service

```diff
@@ -5,7 +5,8 @@ Documentation="https://github.com/erminas/noderedrevpinodes-server"
 
 [Service]
 Type=simple
-ExecStart=/bin/sh -c 'exec /usr/bin/python3 -u revpi-server.py 2>> revpi-server.log'
+StandardOutput=append:%L/revpi-server.log
```

Comment:
Please note that ``append`` as well as ``file``  is only available on systems with systemd version >= 240. So this unit file **won't** work with the stretch image.

Reply:
@nbuchwitz You are right, this is a good point.